components/header/Header.tsx

```diff
@@ -0,0 +1,78 @@
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import AnimatedPics from '../animated-pics/AnimatedPics';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      marginTop: '1rem',
+      marginBottom: '1rem',
+    },
+    textContainer: {
+      marginLeft: '3rem',
+    },
+    mainText: {
+      margin: '3rem 0 2rem 2rem',
+      fontSize: '2rem',
+      fontWeight: 'bold',
+    },
+    secondaryText: {
+      color: 'rgb(255, 255, 255)',
+      fontWeight: 'bold',
+      fontSize: '1.25rem',
+      lineHeight: '1.5',
+    },
```

Comment:
We are probably not reusing these styles anywhere (I think you also mentioned that). But for the other components it will make sense to put more in the "global theme" and import from there.
Like having default fontSizes (small, medium, large) and colors etc. there.